src/pybind/mgr/dashboard/frontend/src/app/shared/services/auth-storage.service.ts

```diff
@@ -18,6 +18,7 @@ export class AuthStorageService {
   remove() {
     localStorage.removeItem('access_token');
     localStorage.removeItem('dashboard_username');
+    localStorage.removeItem('dashboard_interval');
```

Comment:
Using `AuthStorageService` for storing things unrelated to Auth is a bit tricky, right?

On a completely separate topic: [using `localStorage` is potentially dangerous](https://www.rdegges.com/2018/please-stop-using-local-storage/), as data is persisted until explicitly removed and shared across browser tabs and readable for any other page running in the same browser, and I'm not 100% sure that we have covered all possible login/logout paths. Maybe we need to review how to achieve persistence for user settings beyond the browser.

Reply:
I agree with you.
However, the problem now is if we use 'interval' as a component variable, even if it is a private property, when we reload the page, we will lose the latest interval we set, just return to the initial interval, so until now, only put 'interval' into "localStorage", we won't lose the interval.
I am very happy to hear from anyone and then continue improve my implementation.

Reply:
For my second concern, I'd suggest either using:
- `sessionStorage` instead (it's not shared between browser tabs/windows but keeps persisted on F5/reloading).
- storing/retrieving data from an endpoint (via [Ceph KV store](http://docs.ceph.com/docs/master/mgr/plugins/#kv-store))